eZ/Publish/Core/FieldType/Image/ImageThumbnailStrategy.php

```diff
@@ -43,12 +43,16 @@ public function getFieldTypeIdentifier(): string
 
     public function getThumbnail(Field $field, ?APIVersionInfo $versionInfo = null): ?Thumbnail
     {
-        /** @var \eZ\Publish\SPI\Variation\Values\ImageVariation $variation */
-        $variation = $this->variationHandler->getVariation(
-            $field,
-            $versionInfo ?? new VersionInfo(),
-            $this->variationName
-        );
+        try {
+            /** @var \eZ\Publish\SPI\Variation\Values\ImageVariation $variation */
+            $variation = $this->variationHandler->getVariation(
+                $field,
+                $versionInfo ?? new VersionInfo(),
+                $this->variationName
+            );
+        } catch (\Exception $e) {
```

Comment:
Missing import.

Reply:
> What's wrong with `\Exception $e` though? We have plenty of such notations in the kernel :(

It's a matter of convention. We don't perform bulk replace for the old codebase, but try to remember about those rules for the new code.